packages/api/amplify_api/test/graphql_test.dart

```diff
@@ -309,18 +305,12 @@ void main() {
         pollInterval: Duration(seconds: 1),
       );
 
-      fakePlatform = MockConnectivityPlatform();
-      ConnectivityPlatform.instance = fakePlatform;
-      final connectivity = Connectivity();
-      mockClient = MockPollClient();
-
       mockWebSocketBloc = MockWebSocketBloc(
         config: testApiKeyConfig,
         authProviderRepo: getTestAuthProviderRepo(),
         wsService: mockWebSocketService!,
         subscriptionOptions: subscriptionOptions,
         pollClientOverride: mockClient.client,
-        connectivityOverride: connectivity,
```

Comment:
I guess this answers my question as to when `connectivityStreamCreator` i null. Although I would prefer to not have it nullable as we wouldn't use the bloc realistically without it.

Reply:
I made this non-null but I think there's an important note here that the bloc can be used without connectivity info as we would expect a non-Flutter user to not necessarily provide that info. Although the "no info mode" is now a default empty stream rather than a null provider as it was originally, the idea is still the same. For that reason I've intentionally passed the default provider in some of the tests to ensure it works without network connectivity info.